Makefile

```diff
@@ -40,6 +40,11 @@ lint: ## Run linter
 	@echo "==> Linting all packages..."
 	golangci-lint run $(SOURCE_FILES)
 
+.PHONY: fix-lint
```

Comment:
I am not sure what it is the purpose of this line.

Reply:
It's a make thing, helps it avoid being confused by there being an actual file named like this, or something like it